pkg/ui/react-app/src/App.css

```diff
@@ -0,0 +1,253 @@
+.panel {
+  margin-bottom: 20px;
+}
+
+input[type='checkbox']:checked + label {
+  color: #286090;
+}
+
+.custom-control-label {
+  cursor: pointer;
+}
+
+.togglers-wrapper .form-group {
+  margin-bottom: 0.5rem;
+}
+
+[for$='-toggler'].custom-control-label::before,
+[for$='-toggler'].custom-control-label::after {
+  top: 0.28rem;
+  left: -1.3rem;
+  width: 1.12rem;
+  height: 1.12rem;
+}
+
+.capitalize-title::first-letter {
+  text-transform: capitalize;
+}
+
+.expression-input {
+  margin-bottom: 10px;
+}
+
+.expression-input textarea {
+  /* font-family: Menlo,Monaco,Consolas,'Courier New',monospace; */
+  resize: none;
+  overflow: hidden;
+}
+
+button.execute-btn {
+  width: 84px;
+}
+
+.alert.alert-danger {
+  margin-bottom: 10px;
+}
+
+.nav-tabs .nav-link {
+  cursor: pointer;
+}
+
+.tab-content {
+  border-left: 1px solid #dee2e6;
+  border-right: 1px solid #dee2e6;
+  border-bottom: 1px solid #dee2e6;
+  padding: 10px;
+}
+
+.tab-content .alert {
+  margin-bottom: 0;
+}
+
+.data-table.table {
+  margin: 10px 0 2px 0;
+}
+
+.data-table > tbody > tr > td {
+  padding: 5px 0 5px 8px;
+  font-size: 0.8em;
+  overflow: hidden;
+}
+
+.autosuggest-dropdown {
+  position: absolute;
+  border: 1px solid #ced4da;
+  background-color: #fff;
+  color: #495057;
+  font-size: 1rem;
+  z-index: 1000;
+  left: 56px;
+  margin-top: -6px;
+}
+
+.autosuggest-dropdown-list {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.autosuggest-dropdown-list li {
+  width: 100%;
+  padding: 0.25rem 1.5rem;
+  clear: both;
+  white-space: nowrap;
+  background-color: transparent;
+  border: 0;
+  display: block;
+}
+
+.autosuggest-dropdown-list li.autosuggest-dropdown-header {
+  background-color: #bfdeff;
+  font-size: 10px;
+  line-height: 1.5;
+  text-transform: uppercase;
+  text-align: center;
+}
+
+.graph-controls,
+.table-controls {
+  margin-bottom: 10px;
+}
+
+.graph-controls input,
+.table-controls input {
+  text-align: center;
+}
+
+.graph-controls .range-input input {
+  width: 50px;
+}
+
+.time-input input {
+  border-right: none;
+}
+
+.time-input {
+  width: 270px !important;
```

Comment:
Is there a good reason to use `!important` here?

Reply:
This is taken from upstream as it is.

Reply:
Indeed: https://github.com/prometheus/prometheus/blob/d996ba20ec9c7f1808823a047ed9d5ce96be3d8f/web/ui/react-app/src/App.css#L126